app/tests/spec/lib/metrics.js

```diff
@@ -196,7 +198,7 @@ function (chai, $, p, Metrics, AuthErrors, Environment, sinon, _, WindowMock, Te
               assert.equal(windowMock.navigator.sendBeacon.getCall(0).args[0], '/metrics');
 
               var data = JSON.parse(windowMock.navigator.sendBeacon.getCall(0).args[1]);
-              assert.lengthOf(Object.keys(data), 22);
+              assert.lengthOf(Object.keys(data), 23);
```

Comment:
Seeing as this test has assertions for all of the other values, is it potentially worth adding an `assert.isNumber(data.startTime)`, just for the sake of completeness?